tools/pipelines/templates/build-npm-package.yml

```diff
@@ -305,6 +314,8 @@ stages:
                 mkdir $(Build.ArtifactStagingDirectory)/pack/
                 mkdir $(Build.ArtifactStagingDirectory)/pack/scoped/
                 mkdir $(Build.ArtifactStagingDirectory)/test-files/
+                mkdir $(Build.ArtifactStagingDirectory)/log/
+                mv ${{ parameters.buildDirectory }}/packages/test/test-end-to-end-tests/tinylicious.log $(Build.ArtifactStagingDirectory)/log/
```

Comment:
@curtisman @anthony-murphy  Do you think we need to wrap this relative path as an additional path variable?